go.mod

```diff
@@ -10,7 +10,9 @@ require (
 )
 
 require (
+	github.com/cpuguy83/go-md2man/v2 v2.0.2 // indirect
```

Comment:
Did we still want to push this up a nudge, to v2.0.3? https://github.com/cpuguy83/go-md2man/releases/tag/v2.0.3

I should also check with @cpuguy83 if he's planning to do v2.0.4, as some fixes were merged; https://github.com/cpuguy83/go-md2man/compare/v2.0.3...c2c0656178869720a1d9b013ed881db8b7a191ae

Reply:
Can we do this as follow-up?

Reply:
Ah, yes, sure; just recalled we were discussing it last week or so 😂